snapcraft/commands/lint.py

```diff
@@ -309,7 +309,7 @@ def _install_snap(
         is_dangerous = not bool(assert_file)
 
         if assert_file:
-            ack_command = snap_cmd.formulate_ack_command(assert_file)
+            ack_command = snap_cmd.formulate_ack_command(PurePosixPath(assert_file))
```

Comment:
I think it's because of [this](https://github.com/canonical/craft-providers/commit/832784819045ca5b18240d74eacf93f5bb482a7c#diff-1bd0de778fc122dd03546493eca9d2496c4c24df86aa559c8497f27229ef4d7dR24) change, which came in craft-providers 1.14.0 but I guess we skipped that version in snapcraft

Reply:
actually do you know if there's a more "correct" way to create a `PurePosixPath` from a `Path`? Would `PurePosixPath(assert_file.as_posix())` be better?